### ISSUE TITLE
git add: Access is denied.

### ISSUE THREAD

**Issue by maxwellhertz** [src-d/go-git#1155]
The OS is windows 10. I wanted to `git add` with following code:
```go
_, err := w.Add(filename)
```
However, an error occurred: 
```
rename file: Access is denied.
```
I am new here. Could anyone help?

**Comment by xoac** [comment]
I struggle the same error. Implementation just works fine on Linux. And manually works on Windows. (Using git bash).

@maxwellhertz  Did you resolve your problem?

**Comment by maxwellhertz** [comment]
> I struggle the same error. Implementation just works fine on Linux. And manually works on Windows. (Using git bash).
> 
> @maxwellhertz Did you resolve your problem?

No, I just gave up.:joy: At first I wanted to write a program to automate my everyday git stuff but I couldn't turn this problem around. So I changed my mind and started using Github desktop instead.:sweat_smile:
